sound/hybrid_font.h

```diff
@@ -484,12 +484,12 @@ class HybridFont : public SaberBase, public Looper {
     bool most_blades = location.on_blade(0);
 #ifdef ENABLE_IDLE_SOUND
     if (most_blades) {
-      Effect* idle = AvoidIdleSDAccess() ? nullptr : &SFX_idle;
+      Effect* bgnidle = AvoidIdleSDAccess() ? nullptr : &SFX_bgnidle;
       if (SFX_pstoff) {
         SFX_in.SetFollowing(&SFX_pstoff);
-        SFX_pstoff.SetFollowing(idle);
+        SFX_pstoff.SetFollowing(SFX_bgnidle ? bgnidle : &SFX_idle);
```

Comment:
If I remember correctly, I fixed it so that if a "following" effect doesn't exist, it automatically goes to the next one. So it should be fine to just always link pstoff to bgnidle.

Reply:
Cool, yes it does work.
I also had to add boot handling and add bgnidle to StopIdleSound().